shopify-graphicsmagick.rb

```diff
@@ -0,0 +1,109 @@
+# This file was copied from homebrew/homebrew:graphicsmagick.rb, changes include:
+# * url and sha256 changed to newer tarball
+# * conflict with graphicsmagick added
+# * bottle stanza updated
+class ShopifyGraphicsmagick < Formula
+  desc "Image processing tools collection"
+  homepage "http://www.graphicsmagick.org/"
+  url "http://apt.shopify.com/dist/graphicsmagick/GraphicsMagick-1.4.020150705.tar.bz2"
+  sha256 "334fa009c7a1b5b91849b971ec85dffff19fb23737568cfdb49aee853731e7a5"
+  revision 1
+
+  bottle do
+    root_url "http://burkelibbey.s3.amazonaws.com"
+    sha256 "754fb38cc5eb8eff944377d78e000177438bc76db9937675c741364e39f77bd3" => :el_capitan
+  end
+
+  conflicts_with "graphicsmagick", :because => "shopify-graphicsmagick is newer"
```

Comment:
From what I can tell the build starts with the 1.3.12 version and then applies the source for 1.4.020140823